data/persons.txt

```diff
@@ -1850,3 +1850,200 @@ ship "Modified Ladybug"
+			"What do you mean you don't have a save file for this pull request!?"
+			"This statement is, intentionally wrote wrongly to make point."
+
+#		Saugia
+			"Sometimes, a large amount of detail on a model can look less appealing."
+			"Sometimes, a small amount of detail on a model can look more appealing."
+			"Make sure to look at an image from a few steps back as well."
+			"Down with tangents, up with overlaps."
+			"Don't be afraid of adding more contrast."
+			"If you want to learn how to create artwork, start with a donut."
+
+#		tibetiroka
+			"It's just the Steam CI, no need to panic."
+			"Did you check for nullptr?"
+			"		How do we indent after line breaks again?"
+			"Please put the includes in alphabetical order."
+			"Always keep three empty lines between functions."
+			"Why do aliens speak English?"
+
+#		bene-dictator
+			"Just a syntax review. Hold on, I have to rewrite this whole paragraph for you quickly. Delay the merge!"
+			"I was requested for spelling? Oh my goodness, this needs a rewrite..."
+			"One more review. Pretty please?"
+			"Just one more review before merging."
+			"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
+			"Comma here. Comma doesn't go there. I think that one needs a comma. Hold on, what was a comma again?"
+			"Unidentified sentence appears to be lacking a comma. Very dangerous, and highly likely to incite multiple suggestions on where the comma should go. Calling backup reviewers."
+			"Commas are your greatest friends."
+			"Commas are your greatest enemies."
+
+#		roadrunner56
+			"More semicolons!"
+			"...I think I broke something."
+			"Canadian spelling fails me again."
+			"This is my last review, I swear."
+			"Looks like school's starting back up again. See you in a few months."
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request not to be merged"
+			"If you prefer that your pull request never be merged"
+			"If you prefer that your pull request sits idle"
+			"If you want your pull request to be closed"
+		word
+			", "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
```

Comment:
```suggestion

	phrase
		word
			"You don't have to be a reviewer to review pull requests"
			"Anyone can review a pull request"
			"If you don't know how to help, try reviewing some pull requests"
			"Suggestions are welcome on pull requests even if you aren't a reviewer"
			"Oh! I found a typo here but I'm not a reviewer, but you don't need to be a reviewer to review pull requests"
			"Oh! I have an idea of how to rewrite this paragraph; I'll review even though I'm not a reviewer."
		word
			"."
			"!"
```

Reply:
I can't suggest on your suggestion, so I'll comment it:

> "Oh! I have an idea of how to rewrite this paragraph, so I'll review"

Shouldn't that be `an idea on`?

Reply:
Yep, it should be. Editing...